src/NServiceBus.Core/obsoletes.cs

```diff
@@ -1346,7 +1346,7 @@ public interface ISendOnlyBus : IDisposable
     [ObsoleteEx(
         TreatAsErrorFromVersion = "6",
         RemoveInVersion = "7",
-        Message = "IHandleMessages<T> now exposes the IMessageHandlerContext parameter. You can use this to access what used to be available in the IBus interface.  If you're trying to send messages outside IHandleMessages<T> use either the provided IBusSession or create one with the IBusSessionFactory.")]
+        Message = "IHandleMessages<T> now exposes the IMessageHandlerContext parameter. You can use this to access what used to be available in the IBus interface.  If you're trying to send messages outside IHandleMessages<T> use either the provided IMessageSession or create one with the IBusSessionFactory.")]
```

Comment:
Not your fault, but `IBusSessionFactory` is no longer available. Can you remove this from the obsolete message?